packages/account/src/Components/file-uploader-container/file-uploader.tsx

```diff
@@ -77,7 +85,9 @@ const FileUploader = React.forwardRef(({ onFileDrop, getSocket }, ref) => {
                     }
 
                     // send files
-                    const uploader_promise = uploader.upload(processed_files[0]).then(api_response => api_response);
+                    const uploader_promise = uploader
+                        .upload(processed_files[0])
+                        .then((api_response: unknown) => api_response);
```

Comment:
```suggestion
                        .then(<T>(api_response: T) => api_response);
```

What about using generic instead of unknown?
sorry I don't have the deriv-app source right now could u pls check this too :pray: